src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs

```diff
@@ -2567,6 +2649,43 @@ private static ulong ExtractFractionAndBiasedExponent(double value, out int expo
             return fraction;
         }
 
+        private static unsafe ushort HalfToUInt16Bits(Half value)
```

Comment:
This should probably be on `BitConverter`, with the others, and there should likely be an API proposal to expose it (although probably as `Int16Bits` for parity).